src/gleam/float.gleam

```diff
@@ -228,6 +228,21 @@ pub fn absolute_value(float: Float) -> Float {
   }
 }
 
+/// Returns the absolute distance of the inputs as a positive Float
+///
+/// ## Examples
+///
+///    > distance(-10.0, 10.0)
+///    20.0
+///
+///    > distance(0.0, -2.0)
+///    2.0
+///
+pub fn distance(a: Float, b: Float) -> Float {
```

Comment:
This isn't a function I'm familiar with, it is commonly found in other standard libraries?

Reply:
I noticed that I use the same pattern over and over again and extracted it into a function and then discovered that that thing exists in math (who would have thought ;-).

> This isn't a function I'm familiar with

I am going to rename it to what it is called in math AFAIU:
https://en.wikipedia.org/wiki/Absolute_difference

> it is commonly found in other standard libraries?

All I could find:

* Javascript Question https://stackoverflow.com/a/3156794
* Python Question: https://stackoverflow.com/a/13602196
* C++ Question https://stackoverflow.com/a/10589599
* Ruby Question https://stackoverflow.com/a/48577755
* PHP Question https://stackoverflow.com/a/39972873

Now one can argue this is easily solvable with `abs()` or alike, and it is.

I value these kinds of functions because they allow to pipe (where for regular operators I need to create an anon function in place).

However to keep the API surface for the stdlib small they may be better of in some community math.int and math.float libraries maybe? If you want me to I can make remove them or make them local anon functions?

Reply:
Removed `absolute_difference()`. If you still want it I can put it in a separate PR.